cohorts/load.py

```diff
@@ -299,7 +317,7 @@ def plot_survival(self, on, col=None, col_equals=None, how="os", threshold=None)
         results = plot_kmf(
             df=df,
             condition_col=plot_col,
-            censor_col=self.dead_col if how == "os" else self.progressed_col,
+            censor_col=self.dead_col if how == "os" else self.progressed_or_dead_col,
```

Comment:
Added an optional parameter for `progressed_or_dead` and check it against `dead | progressed` if it's passed in.